t/33-developer_mode.t

```diff
@@ -80,6 +80,23 @@ $ws = create_websocket_server(undef, 0, 0);
 $scheduler = create_scheduler;
 $livehandler = create_live_view_handler;
 
+# logs out and logs in again as the specified user; tries multiple times to workaround poo#128807
+my $max_login_attempts = $ENV{OPENQA_DEVEL_MODE_TEST_MAX_LOGIN_ATTEMPTS} // 10;
+sub relogin_as ($user) {
+    my $login_text = '';
+    my $expected_login_text = 'Logged in as ' . $user;
+    for (my $attempts = 0; $attempts < $max_login_attempts; ++$attempts) {
+        if ($login_text ne 'Login') {
+            $driver->get('/logout');
+            $driver->element_text_is('#user-action a', 'Login', 'logged-out before logging in as ' . $user);
+        }
+        $driver->get('/login?user=' . $user);
```

Comment:
that's unexpected. Why is not covered now?

Reply:
I also don't understand this. If it was not covered we would never execute `return pass` which would result in a test failure. I suspect the coverage tracking is broken.